ActindoWorkFlow/UserInterface/02a_Workflows.md

```diff
@@ -226,60 +152,91 @@ The list displays all workflow versions of the selected workflow. Depending on t
 
 - Transition  
     Building blocks that are chained together through places to configure a workflow, represented by a rectangle. All transitions contain a certain, predefined number of input and output ports, represented by a small square to the left and the right of the transition.   
-    When clicking a transition in the workflow editor, it becomes highlighted and the corresponding element settings are displayed in the settings side bar.
+    When clicking a transition in the workflow editor, it becomes highlighted and the corresponding element settings are displayed in the settings sidebar.
 
-    ![Settings side bar](../../Assets/Screenshots/ActindoWorkFlow/Workflows/SettingsSideBar02.png "[Settings side bar]")
+    ![Settings sidebar](../../Assets/Screenshots/ActindoWorkFlow/Workflows/SettingsSideBar02.png "[Settings sidebar]")
 
-    The transition settings side bar is located to the right of the workspace and allows to define the transition settings. The following fields are displayed:
+    The transition settings sidebar is located to the right of the workspace and allows to define the transition settings. The following fields are displayed:
 
     - *Key*  
-        Transition key. This field is read-only. The key is required for API access and must be unique within the workflow version.
+        Transition key. This field is read-only. The key is required for API access and must be unique within the workflow version.<!--- Was ist damit gemeint? und wenn ich eine transition mehrfach einfüge?...-->
 
     - *Label*  
-        Transition label defined by the user. It can be used to specify or simplify the transition name. When editing the transition label, it is changed both at the top of the settings side bar and in the transition rectangle in the diagram.
+        Transition label defined by the user. It can be used to specify or simplify the transition name. When editing the transition label, it is changed both at the top of the settings sidebar and in the transition rectangle in the diagram.
 
     - *Queue type*  
-        Click the drop-down list to select the applicable queue type.   
+        Click the drop-down list to select the applicable queue type, see [Queue types](05_QueueTypes.md).
+
+    - *Priority*   
+        The *Priority* field is used in high performance scenarios. You can use it to speed up a critical path when there are so many actions to be processed that they cannot be processed in parallel, and they pile up in the backlog. You do this by prioritizing the actions that are in the critical path. For example, a critical path might be when an order has been received and needs to be sent to shipping, while other actions such as sending emails are not as time-critical. Note: Priority 10 increases the processing probability by a factor of 10 over priority 1, and priority 8 increases the processing probability by a factor of 4 over priority 2.
+
```

Comment:
Ich verstehe nicht ganz, was es in der Note gemeint ist, es kann aber an fehlenden Kenntnissen legen. 
10 zu 1 = 10 mal so schnell als mit Prio 1? Und 8 zu 2? 4 mal so schnell als mit Prio 2?  Evtl. ein "Fazit" oder allgemeine Regel?